src/devices/Lcm1602a1/IGpioController.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+
+namespace Iot.Device.CharacterLCD
+{
+    // Can we move this upstream and actually derive from this? Should I2C/SPI derive from GpioDriver instead?
+    // Having an interface would allow for easier unit testing...
+    public interface IGpioController : IDisposable
```

Comment:
This work is tracked by #125. I think this is the wrong place for this, and should instead be on the main library. Do you mind moving it up to System.Device.GPIO instead? That way we can also have the GpioController implement the interface and you won't need the GpioControllerAdapter class anymore.

Reply:
Not a problem- didn't want to block on it to get working.